test/test_runner/test_parser.cpp

```diff
@@ -351,18 +353,24 @@ void TestParser::tokenize() {
     }
 }
 
-bool TestParser::extractConnName(std::string& query, TestStatement* statement) {
+void TestParser::extractConnName(std::string& query, TestStatement* statement) {
     std::regex pattern(R"(\[(conn.*?)\]\s*(.*))");
     std::smatch matches;
     bool statement_status = false;
     if (std::regex_search(query, matches, pattern)) {
         if (matches.size() == 3) {
             statement_status = true;
-            statement->conn_name = matches[1];
+            statement->connName = matches[1];
             query = matches[2];
         }
     }
-    return statement_status;
+
+    auto& caseStatus = testGroup->multipleConns;
```

Comment:
So you are assigning default connection name when `multipleConn` is enabled and a `connName` can not be found through regex matching. I wonder if we can simplify the logic as 

```
if connName not found through regex
     statement->connName = DEFAULT
```